### PR TITLE
Add message variation in storybook

### PR DESCRIPTION
**Pull request by NeenaDev**
Fix:https://github.com/vanilla/knowledge/issues/1417

Add message variations into storybook:
- message with short title.
- message with long title.
- message with title and icon.
- message with title and no  icon.

Add pageTitleStyles.tsx

### CODE REVIEW

**Comment by charrondev** [comment]
Something seems wrong here:

image


From the mockups this is supposed to be aligned to the left when there is no icon

image

File: library/src/scripts/messages/fixedMessage.story.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+    const _fixedMessage = fixedMessageFlag && (
+        <div
+            style={{
+                height: unit(titleBarVariables().sizing.height),
+                position: "relative",
+                marginTop: negative(unit(titleBarVariables().sizing.height)),
+            }}
+        >
+            <Message
```
Comment (slafleche) [inline library/src/scripts/messages/fixedMessage.story.tsx]: I'd like to see the title bar above the fixed message in this story.

File: library/src/scripts/messages/messageStyles.ts [hunk 2]
```diff
@@ -210,16 +215,20 @@ export const messagesClasses = useThemeCache(() => {
     });
 
     const messageIcon = style("messageIcon", {
-        ...absolutePosition.middleLeftOfParent(),
+        //...absolutePosition.middleLeftOfParent(),
```
Comment (slafleche) [inline library/src/scripts/messages/messageStyles.ts]: Clean up commented code.

File: library/src/scripts/messages/messageStyles.ts [hunk 3]
```diff
@@ -90,7 +95,7 @@ export const messagesClasses = useThemeCache(() => {
         const padding = noIcon
             ? {
                   vertical: vars.spacing.padding.vertical,
-                  left: vars.spacing.padding.right,
+                  left: vars.spacing.padding.right + 30,
```
Comment (slafleche) [inline library/src/scripts/messages/messageStyles.ts]: Move this to variable.

File: library/src/scripts/messages/messageStyles.ts [hunk 4]
```diff
@@ -48,6 +49,9 @@ export const messagesVariables = useThemeCache(() => {
             fg: globalVars.messageColors.warning.state,
         },
     });
+    const title = themeVars("title", {
+        marginTop: 6,
```
Comment (slafleche) [inline library/src/scripts/messages/messageStyles.ts]: Make variable

File: library/src/scripts/messages/message.story.tsx [hunk 5]
```diff
@@ -18,109 +18,144 @@ import SmartLink from "@library/routing/links/SmartLink";
...
 
 story.add("Message", () => {
     const classesMessages = messagesClasses();
     const [shortMessageFlag, setShortMessageFlag] = useState(true);
+    const [messageWithTitleFlag, setMessageWithTitleFlag] = useState(true);
     const [longMessageFlag, setLongMessageFlag] = useState(true);
-    const [fixedMessageFlag, setFixedMessageFlag] = useState(true);
-    const _shortMessage = shortMessageFlag && (
+    const [iconMessageFlag, setIconMessageFlag] = useState(true);
+    const _attachmentErrorIcon = (
```
Comment (slafleche) [inline library/src/scripts/messages/message.story.tsx]: What do these names starting with "_" mean?

File: library/src/scripts/messages/messageStyles.ts (hunk 2 above)
Comment (NeenaDev) [reply library/src/scripts/messages/messageStyles.ts]: delete commented code lines.

File: library/src/scripts/messages/messageStyles.ts (hunk 3 above)
Comment (NeenaDev) [reply library/src/scripts/messages/messageStyles.ts]: Fixed

File: library/src/scripts/messages/messageStyles.ts (hunk 4 above)
Comment (NeenaDev) [reply library/src/scripts/messages/messageStyles.ts]: Fixed.

File: library/src/scripts/messages/message.story.tsx (hunk 5 above)
Comment (NeenaDev) [reply library/src/scripts/messages/message.story.tsx]: delete this variable

**Pull request merged**